airflow/providers/yandex/__init__.py

```diff
@@ -14,3 +14,17 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from __future__ import annotations
+
```

Comment:
Just wondering this file does not have the note that it was auto-generated. Perhaps is this because of our suspension logic?
Did we create this file manually?

Reply:
> Just wondering this file does not have the note that it was auto-generated. Perhaps is this because of our suspension logic?
> Did we create this file manually?

Good point - I created it with an earlier version og the tool, and since it is excluded provider it is skipped when generating the docs, so we need to manually bump it manually. This is not a problem even if we don't in the future because when it gets unsuspended and prepared for next release it will be overwritten anyway, but I wanted to make sure we have everything generated anyway. Good catch!

Reply:
Fixed. Also added comment.